package.json

```
@@ -12,10 +12,11 @@
   "keywords": [],
   "license": "MIT",
   "dependencies": {
+    "@redocly/portal-plugin-async-api": "^1.0.230",
     "@codemirror/state": "6.4.1",
     "@codemirror/view": "^6.22.2",
     "@lezer/highlight": "^1.2.0",
-    "@redocly/realm": "0.91.3",
+    "@redocly/realm": "^0.92.1",
```

As suggested by Redocly, updating to the latest Realm version
```suggestion
    "@redocly/realm": "^0.97.6",
```